modules/libjpeg_turbo/2.1.4/patches/add_build_file.patch

```diff
@@ -0,0 +1,792 @@
+--- BUILD.bazel
++++ BUILD.bazel
+@@ -0,0 +1,789 @@
++# Copied from https://github.com/tensorflow/tensorflow/blob/master/third_party/jpeg/jpeg.BUILD
++# The copied code is available under the Apache-2.0 license
++# Copyright of the copied code belongs to the TensorFlow Authors
++
++# Description:
++#   libjpeg-turbo is a drop in replacement for jpeglib optimized with SIMD.
++
++load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
++
++WIN_COPTS = [
++    "/Ox",
++    "-DWITH_SIMD",
++    "-wd4996",
++]
++
++libjpegturbo_copts = select({
++    ":android": [
++        "-O3",
++        "-fPIC",
++        "-w",
++    ],
++    ":windows": WIN_COPTS,
++    "//conditions:default": [
++        "-O3",
++        "-w",
++    ],
++}) + select({
++    ":armeabi-v7a": [
++        "-D__ARM_NEON__",
++        "-DNEON_INTRINSICS",
++        "-march=armv7-a",
++        "-mfpu=neon",
++        "-mfloat-abi=softfp",
++        "-fprefetch-loop-arrays",
++    ],
++    ":arm64-v8a": [
++        "-DNEON_INTRINSICS",
++    ],
++    ":linux_ppc64le": [
++        "-mcpu=power8",
++        "-mtune=power8",
++    ],
++    "//conditions:default": [],
++})
++
++cc_library(
++    name = "jpeg",
```

Comment:
Could you add the usual alias target matching the module name? Sorry if it's already there and I missed it.